packages/deployment/upgrade-test/upgrade-test-scripts/agoric-upgrade-10/actions.mjs

```diff
@@ -0,0 +1,348 @@
+  agopsLocation,
+  executeCommand,
+} from '../cliHelper.mjs';
+import {
+  HOME,
+  ATOM_DENOM,
+  GOV1ADDR,
+  GOV2ADDR,
+  GOV3ADDR,
+} from '../constants.mjs';
+
+const acceptEC = async () => {
+  console.log('ACTIONS Tickling the wallets so they are revived');
+
+  const govAccounts = [GOV1ADDR, GOV2ADDR, GOV3ADDR];
+  let voter = 0;
+  for (const account of govAccounts) {
+    const runs = [1, 2];
+    for (const run of runs) {
+      console.log(`${account}: ${run}: Accepting EC Committee`);
+
+      if (run === 1) {
+        try {
+          const params = ['ec', 'committee', '--send-from', account];
+          await executeCommand(agopsLocation, params, {
+            timeout: 30000,
+          });
+        } catch (error) {
+          console.warn(error.message);
+        }
+      } else {
+        try {
+          await agops.ec('committee', '--send-from', account, '--voter', voter);
+        } catch (error) {
+          console.warn(error.message);
+        }
+
+        voter += 1;
+      }
+
+      await waitForBlock(3);
+    }
+
+    console.log(`${account}: Accepting EC Charter`);
+    try {
+      await agops.ec('charter', '--send-from', account);
+    } catch (error) {
+      console.warn(error.message);
+    }
+  }
+};
+
+const acceptOracles = async oracles => {
+  for (const oracle of oracles) {
+    console.log(`${oracle.address}: Accept oracle invitations`);
+
+    const offerFunction = async () => {
+      return agops.oracle('accept', '--offerId', oracle.id);
+    };
+
+    await executeOffer(oracle.address, offerFunction);
+  }
+};
+
+const proposeNewAuctionParams = async (
+  address,
+  startFequency,
+  clockStep,
+  priceLockPeriod,
+) => {
+  const offerFunction = async () => {
+    const charterAcceptOfferId = await agops.ec(
+      'find-continuing-id',
+      '--for',
+      `${'charter\\ member\\ invitation'}`,
+      '--from',
+      address,
+    );
+
+    return agops.auctioneer(
+      'proposeParamChange',
+      '--charterAcceptOfferId',
+      charterAcceptOfferId,
+      '--start-frequency',
+      startFequency,
+      '--clock-step',
+      clockStep,
+      '--price-lock-period',
+      priceLockPeriod,
+    );
+  };
+
+  await executeOffer(address, offerFunction);
+};
+
+const voteForNewParams = async position => {
+  const govAccounts = [GOV1ADDR, GOV2ADDR, GOV3ADDR];
+  for (const account of govAccounts) {
+    await agops.ec('vote', '--forPosition', position, '--send-from', account);
+  }
+};
+
+const paramChangeOfferGeneration = async (
+  previousOfferId,
+  voteDur,
+  debtLimit,
+) => {
+  const ISTunit = 1_000_000n; // aka displayInfo: { decimalPlaces: 6 }
+  const voteDurSec = BigInt(voteDur);
+  const debtLimitValue = BigInt(debtLimit) * ISTunit;
+  const toSec = ms => BigInt(Math.round(ms / 1000));
+
+  const id = `propose-${Date.now()}`;
+  const deadline = toSec(Date.now()) + voteDurSec;
+
+  const zip = (xs, ys) => xs.map((x, i) => [x, ys[i]]);
+  const fromSmallCapsEntries = txt => {
```

Comment:
all the smallcaps manipulations in the JS thingy I wrote are to avoid using anything outside the node standard library. Since we've now crossed over into using `package.json` and such, this should switch to using `@endo/marshal` in the normal way.